### DISCUSSION TITLE
待上传和整理的文件

### DISCUSSION THREAD

**Discussion by YouXam** [byrdocs/byrdocs-archive#8] [General]
- [x] https://github.com/faranten/bupt-cs-gallery (没有整理质量太差的文件，包括对试卷进行拍照得到的照片和打印店的大部分文件)
- [x] https://github.com/Simon9997/BUPT-final-exam
- [x] https://github.com/William-Seven/Collected-study-materials-from-BUPT
- [x] https://github.com/MsunM-cell/Third-Year
- [x] https://github.com/Cowboy-Spike-Spiegel/BUPT_SCS_Resources
- [ ] https://gitee.com/guanxidazhuang/bupter_examination_part1
- [ ] https://gitee.com/guanxidazhuang/bupter_examination_part2
- [ ] https://gitee.com/guanxidazhuang/bupter_examination_part3

有其他相关的欢迎同学们补充。

如果想参与文件整理和贡献，请参阅[贡献指南](https://github.com/byrdocs/byrdocs-archive/blob/master/CONTRIBUTING.md)。

如果你有成体系的整理计划，请将计划发在此讨论串。

**Comment by cppHusky** [comment]
- [x] [faranten/bupt-cs-gallery/计算机组成原理/往年题](https://github.com/faranten/bupt-cs-gallery/tree/main/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BB%84%E6%88%90%E5%8E%9F%E7%90%86/%E5%BE%80%E5%B9%B4%E9%A2%98)

**Comment by YouXam** [comment]
- [x] [faranten/bupt-cs-gallery](https://github.com/faranten/bupt-cs-gallery) _注：没有整理质量太差的文件，包括对试卷进行拍照得到的照片和打印店的大部分文件_

**Comment by EliorFoy** [comment]
- [x] faranten/bupt-cs-gallery/概率论与数理统计/往年题

**Comment by cppHusky** [comment]
- [x] [Cowboy-Spike-Spiegel/BUPT_SCS_Resources](https://github.com/Cowboy-Spike-Spiegel/BUPT_SCS_Resources)
    - [x] ~~高等数学（下）~~
    - [x] 高等数学（上）
    - [x] 软件工程
    - [x] ~~计算机网络~~
    - [x] ~~计算机组成原理~~
    - [x] 计算机系统结构
    - [x] 编译原理
    - [x] 线性代数
    - [x] ~~算法设计与分析~~
    - [x] 离散数学（下）
    - [x] 离散数学（上）
    - [x] 电路分析与电子学基础
    - [x] ~~现代交换原理~~
    - [x] ~~概率论与数理统计~~
    - [x] 数据结构
    - [x] 数据库系统概念
    - [x] ~~数字逻辑~~
    - [x] 操作系统
    - [x] 形式语言与自动机
    - [x] 大学物理C
    - [x] ~~军事理论~~
    - [x] Linux

**Comment by catYumor** [comment]
- [x] [Simon9997/BUPT-final-exam/大三-Computer-Science-and-Technology/大三](https://github.com/Simon9997/BUPT-final-exam/tree/main/%E5%A4%A7%E4%B8%89-Computer-Science-and-Technology/%E5%A4%A7%E4%B8%89)

**Comment by lumoes** [comment]
- [ ]  https://gitee.com/guanxidazhuang/bupter_examination_part1

**Comment by cppHusky** [comment]
本项目已有相当内容存量，已改变内容收集、管理方针，不再主动新增此类二次整理文件。
